public/application.js

```diff
@@ -1,50 +1,72 @@
 $(document).ready(() => {
   play();
+  browserBackButtonClicked();
 });
 
-function play() {
-  $("td").click(e => {
-    e.preventDefault();
-    let request = $.ajax({
+function browserBackButtonClicked(){
+
+  if (window.history && window.history.pushState) {
+
+    window.history.pushState(null, null, '/game');
+    $(window).on('popstate', function() {
+        rewind();
+    });
+  };
+};
+
+function rewind(){
+let request = $.ajax({
         type: "PUT",
-        url: "/game",
-        data: {
-            space: $(e.currentTarget).attr('id'),
-        },
+        url: "/rewind",
         dataType: "json"
     });
-    request.then((response) => {
-      putMarkersDown(response);
-      if (response.over === true) {
-        gameResult(response);
-      };
+    request.then(() => {
+        location.reload();
     });
-  });
+};
+
+function play() {
+$("td").click(e => {
```

Comment:
lets avoid finding an element by html element. finding elements by html elements, id, or class is a big no no. You want to use data-ids. update this to do something like `#("[data-id=something]").click`

Reply:
to elaborate on why only use data ids. elements/ids/class are used by designers, and sometimes designers make changes to their names. we want to avoid a conflict where our javascript code would depend on changes the designers make. data-id are only used by javascript, so its a good identifier for jquery.

Reply:
please update all your jquery's to look for data-ids only.